app/constants/locale.js

```diff
@@ -0,0 +1,5 @@
+export const CONVERT_ISO_TO_IETF = {
```

Comment:
if this is only being used in the languages.js file, what do you think about inlining it to that file?

Reply:
fixed!